telnyx_rtc/build.gradle

```diff
@@ -14,7 +14,7 @@ apply plugin: 'maven-publish'
 android {
     namespace 'com.telnyx.webrtc.sdk.telnyx_rtc'
 }
-
+ 01
 def getVersionName = { ->
     return "1.4.4"
```

Comment:
I forgot to move this to 1.5.0

But can you bump this to 1.6.0 (updating the middle version as we are adding a [backwards compatible feature](https://semver.org/))

And also add to the CHANGELOG.md @isaacakakpo1